frontend/src/utils/data.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Strips empty, undefined, and null fields from an object. This method is
+ * intentionally NOT recursive.
+ *
+ * @example setEmptyFieldsToUndefined({
+ *   a: "",
+ *   b: null,
+ *   c: 123
+ * })
+ * // returns { c: 123 }
+ */
```

Comment:
**suggestion:** turn this into a unit test. It will serve as an example AND verify that `setEmptyFieldsToUndefined` does what it's supposed to be.

Reply:
Good idea, and done.